src/AutoMapper/Configuration/PathConfigurationExpression.cs

```diff
@@ -28,6 +28,11 @@ public void MapFrom<TSourceMember>(Expression<Func<TSource, TSourceMember>> sour
             MapFromUntyped(sourceExpression);
         }
 
+        public void Ignore()
+        {
+            PathMapActions.Add(pm => pm.Ignored = true);
```

Comment:
That might take a bit more refactoring I think, since that method returns an expression and the PropertyMap piece above in `CreateAssignmentFunc` does a similar filter.